src/main/java/org/fcrepo/importexport/importer/Importer.java

```diff
@@ -574,10 +573,8 @@ private Model sanitize(final Model model) throws IOException, FcrepoOperationFai
      * @return true if the resource represents a type that may not be added/removed explicitly
      */
     private boolean forbiddenType(final Resource resource) {
-         return resource.getNameSpace().equals(REPOSITORY_NAMESPACE)
-             || resource.getURI().equals(CONTAINER.getURI())
-             || resource.getURI().equals(NON_RDF_SOURCE.getURI())
-             || resource.getURI().equals(RDF_SOURCE.getURI());
+        return resource.getNameSpace().equals(REPOSITORY_NAMESPACE)
```

Comment:
https://wiki.duraspace.org/display/FEDORA5x/What+are+the+Server+Managed+Triples states that the memento namespace is also server managed, and importing a resource with `rdf:type <http://mementoweb.org/ns#SomeType>` produces the following error and results in a container being created without any of the exported properties present:
```
ERROR (Importer) Error while importing /Users/bbpennel/git/fcrepo-import-export/export-dir/rest/witht.ttl (409): The http://www.w3.org/1999/02/22-rdf-syntax-ns#type predicate may not take an object in the server managed namespaces (http://mementoweb.org/ns#).
```

Seems like it would make sense to add the third disallowed namespace in here at this point.

Reply:
good catch.